gym_minigrid/envs/pedestrian/TwoLaneRoadEnv.py

```diff
@@ -23,6 +25,7 @@ def __init__(
         self,
         pedAgents: List[PedAgent]=[],
         vehicleAgents: List[Vehicle]=[],
+        trajectoryVehicleAgents: List[TrajectoryVehicle]=[],
```

Comment:
there can be a separate list of trajectory agents, environment does not know what type of vehicle, remove this list